src/api-client.ts

```diff
@@ -149,10 +152,17 @@ export class RevAiApiClient {
     }
 
     async getTranscriptObject(id: string): Promise<RevAiApiTranscript> {
+        const stream = await this.getTranscriptObjectStream(id);
+        return stream.read();
```

Comment:
I think these changes break this function.

When I print the results to console:
```
<Buffer 7b 22 6d 6f 6e 6f 6c 6f 67 75 65 73 22 3a 5b 7b 22 73 70 65 61 6b 65 72 22 3a 30 2c 22 65 6c 65 6d 65 6e 74 73 22 3a 5b 7b 22 74 79 70 65 22 3a 22 74 ... >
```

And I can't access any of the properties. The types still show up in the IDE because we are casting raw bytes to our own type, but the returned object is invalid.

Reply:
:( I thought I had fixed this. I'll look more into it